docs/src/main/sphinx/release/release-365.md

```diff
@@ -0,0 +1,126 @@
+# Release 365 (XX Dec 2021)
+
+## General
+
+* Add support for [`TRUNCATE TABLE`](/sql/truncate). ({issue}`8921`)
+* Add support for time travel queries. ({issue}`8773`)
```

Comment:
Maybe use example of actual syntax instead of saying `time travel queries`? Is that a generally used term?

Reply:
Also is that supported by some connectors already. If not I am not sure it is worth mentioning.

Reply:
That's the common way to refer to that kind of queries across databases that support that kind of feature.

> Also is that supported by some connectors already. If not I am not sure it is worth mentioning.

It's not, but if someone implements their own connector, they can take advantage of the functionality.

Reply:
Can we link to the docs? Can we list what connectors support that?

Reply:
No connectors support that yet, but the capability now exists in the engine for any connector that wants to take advantage of it (including third-party connectors not shipped with Trino)